packages/material-renderers/src/controls/MaterialDateControl.tsx

```diff
@@ -73,17 +80,28 @@ export const MaterialDateControl = (props: ControlProps) => {
     ? errors
     : null;
   const secondFormHelperText = showDescription && !isValid ? errors : null;
+
+  const updateChild = () => {
+    setKey(key + 1);
+  };
+
   const onChange = useMemo(
     () => createOnChangeHandler(path, handleChange, saveFormat),
     [path, handleChange, saveFormat]
   );
 
+  const onBlur = useMemo(
+    () =>
+      createOnBlurHandler(path, handleChange, format, saveFormat, updateChild),
+    [path, handleChange, format, saveFormat, updateChild]
+  );
```

Comment:
This `useMemo` will never work as `updateChild` is a new instance on every `render` call, therefore always breaking the memo.